src/app/components/ContentContainer/ContentContainer.tsx

```diff
@@ -11,13 +10,16 @@ export function ContentContainer({
 }: ContentContainerProps) {
   return (
     <section>
-      <GridContainer
-        className={classNames('px-14 py-20', {
-          'px-32': !align,
-        })}
+      <div
+        className={classNames(
+          'ml-auto mr-auto flex max-w-[87.5rem] flex-col px-10 py-4 sm:px-14 sm:py-20',
```

Comment:
Can you make this the default, or allow us to pass in overriding classes?

Reply:
Updated, thanks!